debezium-core/src/main/java/io/debezium/relational/TableImpl.java

```diff
@@ -51,11 +46,6 @@ public TableId id() {
         return pkColumnNames;
     }
 
-    @Override
```

Comment:
Should the method really be removed? I'd keep it there with the logic of trasnaltion of columns to column names. Just maybe rename it to highlight that it is not getter but expensive method like `extractColumnNames()`

Reply:
Hum, hum, I guess I could do that. In general it shouldn't be invoked, but I guess it's manageable when it's documented properly.